src/app-layer-parser.c

```diff
@@ -1853,8 +1835,6 @@ void AppLayerParserRestoreParserTable(void)
  */
 static int AppLayerParserTest01(void)
 {
-    AppLayerParserBackupParserTable();
```

Comment:
I think so

Reply:
By the way, I am not sure what these `AppLayerParserTest01` really test : they check that `AppLayerParserParse` returns failure for this newly registered `ALPROTO_TEST` as `TestProtocolParser` returns error but it would also return failure without the registration...

Reply:
So, could we remove this ALPROTO_TEST ?